### PR TITLE
Typography Update

### PR DESCRIPTION
**Pull request by aletail**
### Ticket Link
https://werkbotstudios.teamwork.com/#/tasks/27811857

### Testing Steps
- [ ] Test out basic elements, confirm the defaults function as expected
- [ ] Try to override some of the elements, confirm this functions as expected

### CODE REVIEW

**Comment by aletail** [comment]
> Properties have been added here that are not present in framewerk\docs\en\form.md. Maybe add these to the documentation branch?

I will make a new task to update documentation, I think this will be on going as we tweak and iron things out.

File: sass/components/form/components/_select.scss [hunk 1]
```diff
@@ -38,6 +45,9 @@ $component-form-select-properties: $default-component-form-select-properties !de
       appearance: none;
       background-color: getThemeProperty(backgroundColor, $component-form-select-properties);
       border: 1px solid getThemeProperty(borderColor, $component-form-select-properties);
+      color: getThemeProperty(textColorDark, $component-form-select-properties);
```
Comment (tiller1010) [inline sass/components/form/components/_select.scss:48]: This is set to the theme property `textColorDark`, not the select `color` property. Also, this component has a `textColor` property.

File: sass/components/form/components/_text.scss [hunk 2]
```diff
@@ -28,6 +35,9 @@ $component-form-text-properties: $default-component-form-text-properties !defaul
       width: 100%;
       box-sizing: border-box;
       border: 1px solid getThemeProperty(borderColor, $component-form-text-properties);
+      color: getThemeProperty(textColorDark, $component-form-text-properties);
```
Comment (tiller1010) [inline sass/components/form/components/_text.scss:38]: Same thing here, this is a theme property.

File: sass/components/form/components/_text.scss [hunk 3]
```diff
@@ -2,11 +2,18 @@
 $default-component-form-text-properties:(
   borderColor: getThemeProperty(borderColor, $component-form-properties),
   shiftingLabelColor: getThemeProperty(labelColor, $component-form-properties),
+  color: getThemeProperty(textColorDark),
+  fontFamily: getThemeProperty(fontText),
```
Comment (tiller1010) [inline sass/components/form/components/_text.scss:6]: This is unused.

File: sass/components/form/components/_text.scss [hunk 4]
```diff
@@ -28,6 +35,9 @@ $component-form-text-properties: $default-component-form-text-properties !defaul
       width: 100%;
       box-sizing: border-box;
       border: 1px solid getThemeProperty(borderColor, $component-form-text-properties);
+      color: getThemeProperty(textColorDark, $component-form-text-properties);
+      font-family: getThemeProperty(fontText, $component-form-text-properties);
```
Comment (tiller1010) [inline sass/components/form/components/_text.scss:39]: This is the theme property

File: sass/components/form/components/_textarea.scss [hunk 5]
```diff
@@ -28,6 +35,9 @@ $component-form-textarea-properties: $default-component-form-textarea-properties
       width: 100%;
       box-sizing: border-box;
       border: 1px solid getThemeProperty(borderColor, $component-form-textarea-properties);
+      color: getThemeProperty(textColorDark, $component-form-textarea-properties);
+      font-family: getThemeProperty(fontText, $component-form-textarea-properties);
```
Comment (tiller1010) [inline sass/components/form/components/_textarea.scss:39]: Same issues here

**Review by tiller1010 — CHANGES_REQUESTED** [review]
I noticed that some of the form component properties here are unused in favor of the theme properties.

**Comment by aletail** [comment]
@tiller1010 I think I have these straighten out - we may need to re-visit this and update the naming conventions so its easier to keep track of haha

**Comment by aletail** [comment]
> Unrelated, but could we remove the calc() calls added here? https://github.com/werkbot/framewerk/pull/17/files they do not work with the sass builds (they give invalid property values and cause the build to fail), and they are only warnings in the webpack build.

Sure, any idea why they were added? I will remove and test them as a part of this PR

**Review by tiller1010 — APPROVED** [review]
Spectacular!

**Pull request merged**